### PR TITLE
fix send sheet focus handling

### PR DESCRIPTION
**Pull request by skylarbarrera**
Fixes RNBW-2045

## What changed (plus any additional context for devs)
I added proper input focusing, this was broken already but was a lot more noticeable after adding the custom gas sheet

## PoW (screenshots / screen recordings) - updated 
https://cloud.skylarbarrera.com/Screen-Recording-2021-12-16-18-47-14.mp4

## Final checklist
[x] Assigned individual reviewers?
[x] Added labels?

### CODE REVIEW

File: src/screens/SendSheet.js [hunk 1]
```diff
@@ -381,13 +386,18 @@ export default function SendSheet(props) {
...
       }
     },
-    [sendUpdateAssetAmount]
+    [sendUpdateAssetAmount, setLastFocusedInputHandle, assetInputRef]
   );
 
+  const handleCustomGasBlur = useCallback(() => {
+    lastFocusedInputHandle?.current?.focus();
+  }, [lastFocusedInputHandle]);
+
```
Comment (osdnk) [inline src/screens/SendSheet.js:400]: there's no need for this wrapping. Why won't you directly use `lastFocusedInputHandle?.current?.focus`?

**Review by osdnk — APPROVED** [review]

**Comment by christianbaroni** [comment]
couple things

- should refocus last focused input if you open the review sheet and then go back to send. currently doesn't refocus any input
- I've seen this only once before but just saw it again first time I tested this PR. could be a coincidence, however we should see if we can fix this. wasn't touching the screen at all here. stuck in some kind of loop: https://www.dropbox.com/s/ws0wip4zb8ak15y/RPReplay_Final1639687855.mov?dl=0
-- I think I opened the gas modal and then went back to send, and that's when it started going crazy

**Comment by skylarbarrera** [comment]
tagging this with android as the current behavior in develop would ruin a new users first impressions imo

**Review by jinchung — APPROVED** [review]
🌮

**Pull request merged**